sql/search_aggregates_v5.sql

```diff
@@ -0,0 +1,91 @@
+WITH
+  augmented AS (
+  SELECT
+    s.*,
+    sc.element.source AS source,
+    sc.element.count AS count,
+    sc.element.engine AS engine,
+    (
+    SELECT
+      ANY_VALUE(element.version)
+    FROM
+      UNNEST(active_addons.list)
+    WHERE
+      element.addon_id = 'followonsearch@mozilla.com') AS addon_version,
+    CASE
+      WHEN (sc.element.source IN ('searchbar', 'urlbar', 'abouthome', 'newtab', 'contextmenu', 'system', 'activitystream', 'webextension', 'alias') OR sc.element.source IS NULL) THEN 'sap'
+      WHEN STARTS_WITH(sc.element.source, 'in-content:sap:')
+    OR STARTS_WITH(sc.element.source, 'sap:') THEN 'tagged-sap'
+      WHEN STARTS_WITH(sc.element.source, 'in-content:sap-follow-on:') OR STARTS_WITH(sc.element.source,'follow-on:') THEN 'tagged-follow-on'
+      WHEN STARTS_WITH(sc.element.source, 'in-content:organic:') THEN 'organic'
+      ELSE 'unknown'
+    END AS type
+  FROM
+    main_summary_v4 AS s,
+    UNNEST(search_counts.list) AS sc
```

Comment:
this should be unnested in a separate CTE from determining `addon_version` and `type` so that those are calculated once, instead of for each element in `search_counts.list`